R/stats-glm-tidiers.R

```diff
@@ -132,7 +135,8 @@ augment.glm <- function(x,
 #' @family lm tidiers
 #' @seealso [stats::glm()]
 glance.glm <- function(x, ...) {
-  check_for_subclasses(x)
+  check_glm_class(x)
```

Comment:
Can you change the name of this to something more informative? Maybe `check_for_broom_mixed_support()`?